report-ng/app/webpack.config.js

```diff
@@ -37,7 +37,8 @@ module.exports = ({production} = {}, {analyze, tests, hmr, port, host} = {}) =>
                 // https://github.com/aurelia/binding/issues/702
                 // Enforce single aurelia-binding, to avoid v1/v2 duplication due to
                 // out-of-date dependencies on 3rd party aurelia plugins
-                'aurelia-binding': path.resolve(__dirname, 'node_modules/aurelia-binding')
+                'aurelia-binding': path.resolve(__dirname, 'node_modules/aurelia-binding'),
+                'tslib': path.resolve(__dirname, 'node_modules/tslib')
```

Comment:
What solves this?

Reply:
The Apache ECharts are using a different version of `tslib`.
(In this case: v2.3.0 from echarts and v2.3.1 from mdc-component)
This caused a duplicate-package-warning, which was resolved with the extra line
`'tslib': path.resolve(__dirname, 'node_modules/tslib')`.

Reply:
You should add this info at least as a comment above or behind this line.

Reply:
A comment was added.